ssh.go

```diff
@@ -20,6 +20,7 @@ type SSHClient struct {
 	conn         *ssh.Client
 	sess         *ssh.Session
 	user         string
+	identityFile string
```

Comment:
Also include option for looking up passphrase from the environment variables.

Use case:

Running sup from a github action, feeding a private key passphrase from the repository secrets, without the need to hardcode the ENV variable